src/knot/dnssec/zone-events.c

```diff
@@ -53,10 +53,10 @@ static int sign_init(zone_update_t *update, conf_t *conf, zone_sign_flags_t flag
 		ctx->now = adjust_now;
 	}
 
-	// perform nsec3resalt if pending
+	// perform nsec3re_salt if pending
 
-	if (roll_flags & KEY_ROLL_ALLOW_NSEC3RESALT) {
-		r = knot_dnssec_nsec3resalt(ctx, &reschedule->last_nsec3resalt, &reschedule->next_nsec3resalt);
+	if (roll_flags & KEY_ROLL_ALLOW_NSEC3RE_SALT) {
+		r = knot_dnssec_nsec3re_salt(ctx, &reschedule->last_nsec3re_salt, &reschedule->next_nsec3re_salt);
```

Comment:
I had a really hard time parsing this.

Possible parsings:
* `result`
* `res-alt`
* `re-salt`

Based on NSEC3 having SALTs, it seems likely that `re-salt` is the right parsing. Since the code is willing to use `_` in places, it'd be nice if it could be written like this instead of as `resalt`.

That said, I did this last and I've left it as the last commit to make it easier for me to drop it instead of sorting it alphabetically into the series.